django_freeradius/utils.py

```diff
@@ -72,9 +73,10 @@ def prefix_generate_users(prefix, n, password_length):
 
 def generate_pdf(prefix, data):
     template = get_template(BATCH_PDF_TEMPLATE)
-    html = template.render(data)
+    html = HTML(string=template.render(data))
+    css = CSS(filename=os.path.abspath('django_freeradius/static/django-freeradius/css/freeradius.css'))
```

Comment:
Is it a bad practice to import `os` and then use it only once in my code for this feature (line 77). Is it better to create a new variable in settings file (like `BATCH_PDF_TEMPLATE`) and then import it here?

Reply:
this approach doesn't work, I tested it from the web interface of the development server and it raises an error.
it works when running tests with `runtests.py` because it happens that from the root, the path `django_freeradius/static/django-freeradius/css/freeradius.css` is correct, but from `tests/` it's not.

Moreover, if we hardcode the CSS value like that, we won't allow users to change their style.

I found a simpler approach. Get rid of the CSS entirely, we'll use an embedded stylesheet definition in the HTML it's not even really used. Then follow the instructions of my next comment.